generators/chipyard/src/main/scala/iobinders/IOBinders.scala

```diff
@@ -352,6 +353,14 @@ class WithSerialTLIOCells extends OverrideIOBinder({
   }
 })
 
+class WithChipIdPin extends OverrideIOBinder({
```

Comment:
This should be `WithChipIdIOCells` to match the naming convention of other I/Os.